test/units/IexecLayerZeroBridge.t.sol

```diff
@@ -112,9 +148,137 @@ contract IexecLayerZeroBridgeTest is TestHelperOz5 {
         vm.prank(user1);
         iexecLayerZeroBridge.send{value: fee.nativeFee}(sendParam, fee, payable(user1));
 
-        // Verify source state - tokens should be locked in adapter
+        // Verify source state - tokens should be burned
         assertEq(rlcCrosschainToken.balanceOf(user1), INITIAL_BALANCE - TRANSFER_AMOUNT);
     }
 
+    // ============ LEVEL 2 PAUSE TESTS (Send Pause) ============
+
+    function test_PauseSends_EmitsCorrectEvent() public {
+        vm.expectEmit(true, false, false, false);
+        emit SendPaused(pauser);
+
+        vm.prank(pauser);
+        iexecLayerZeroBridge.pauseSend();
+    }
+
+    function test_PauseSends_OnlyPauserRole() public {
+        vm.expectRevert();
+        vm.prank(unauthorizedUser);
+        iexecLayerZeroBridge.pauseSend();
+    }
+
+    function test_PauseSends_BlocksOutgoingOnly() public {
+        // Pause send
+        vm.prank(pauser);
+        iexecLayerZeroBridge.pauseSend();
+
+        // Verify state
+        assertFalse(iexecLayerZeroBridge.paused());
+        assertTrue(iexecLayerZeroBridge.sendPaused());
+
+        // Prepare send parameters
+        (SendParam memory sendParam, MessagingFee memory fee) =
+            TestUtils.prepareSend(iexecLayerZeroBridge, addressToBytes32(user2), TRANSFER_AMOUNT, DEST_EID);
+
+        // Attempt to send tokens - should revert with EnforcedSendPause
+        vm.deal(user1, fee.nativeFee);
+        vm.prank(user1);
+        vm.expectRevert(DualPausableUpgradeable.EnforcedSendPause.selector);
+        iexecLayerZeroBridge.send{value: fee.nativeFee}(sendParam, fee, payable(user1));
+
+        // Verify no tokens were burned
+        assertEq(rlcCrosschainToken.balanceOf(user1), INITIAL_BALANCE);
+    }
+
+    function test_UnpauseSends_RestoresOutgoingTransfers() public {
+        // Pause then unpause send
+        vm.startPrank(pauser);
+        iexecLayerZeroBridge.pauseSend();
+
+        vm.expectEmit(true, false, false, false);
+        emit SendUnpaused(pauser);
+        iexecLayerZeroBridge.unpauseSend();
+        vm.stopPrank();
+
+        // Should now work normally
+        assertFalse(iexecLayerZeroBridge.paused());
+        assertFalse(iexecLayerZeroBridge.sendPaused());
+
+        test_SendToken_WhenOperational();
+    }
+
+    // ============ DUAL PAUSE WORKFLOW TESTS ============
+
+    function test_DualPause_PauseFromSendToFull() public {
```

Comment:
This test and an additional one `test_PauseFromFullToSendOnly` should be inside `DualPausableUPgradeable.t.sol`.